pyomo/repn/tests/cpxlp/test_cpxlp.py

```diff
@@ -209,10 +211,10 @@ def test_var_on_deactivated_block(self):
         baseline_fname, test_fname = self._get_fnames()
         self._cleanup(test_fname)
         model.write(test_fname, format='lp')
-        self.assertFileEqualsBaseline(
+        self.assertTrue(cmp(
             test_fname,
-            baseline_fname,
-            delete=True)
+            baseline_fname),
+            msg="Files %s and %s differ" % (test_fname, baseline_fname))
```

Comment:
Do we need another call to `self._cleanup(test_fname)` here?

Reply:
I removed the one at the beginning of the helper function. The one afterwards is important to make sure the test file is being deleted.